test/titles-option.test.ts

```diff
@@ -0,0 +1,28 @@
+import { assert } from "chai";
+import { exec, getDefaultArgs,getProgramFromFiles, generateSchema, Definition } from "../typescript-json-schema";
+import {capitalCase} from 'case-anything'
+
+describe("out option", () => {
+    it("should have title in capital case", async () => {
+        try {
+            await exec("test/programs/interface-single/main.ts", "MyObject", {
+                ...getDefaultArgs(),
+                titles: true,
+            });
+            const program = getProgramFromFiles(["test/programs/interface-single/main.ts"]);
+            let schema = generateSchema(program, "MyObject", {titles: true}); 
+            assert.notEqual(schema, null)
+            let properties = schema!.properties;
+            assert.notEqual(schema!.properties, undefined)
+            properties = properties!;
+            for (const property of Object.entries(properties)) {
```

Comment:
Instead of testing with some logic, please just create a new test case or update an existing test case with a fully specified example.

Reply:
Which?

Reply:
What’s your question?

Reply:
I do not understand what I should do

Reply:
Unit tests should have as little logic as possible to make them easier to debug and find issues. Your test uses the same library we use to generate the title so if something ever changes, we don’t notice it in the test. Ideally tests don’t have loops or call libraries.